### PR TITLE
[SLAPI-68] Support online payment options

### PR DESCRIPTION
**Pull request by RuyiLi**
# Purpose

Adds support for interacting with the online payment options resource.

# Related Material

See https://www.freshbooks.com/api/online-payments

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 - Move default branch from master to main
 
+### 2.2.0
```
Comment (amcintosh) [inline CHANGELOG.md:9]: Let's just put this in with to 3.0.0 changes

File: packages/api/src/models/PaymentOptions.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+/* eslint-disable @typescript-eslint/camelcase */
+
+export enum PaymentGatewayName {
+	WePay = 'fbpay',
+	Stripe = 'stripe',
+	PayPal = 'paypal',
+}
+
+export interface PaymentOptions {
```
Comment (amcintosh) [inline packages/api/src/models/PaymentOptions.ts:11]: Can we add to https://github.com/freshbooks/freshbooks-nodejs-sdk/blob/main/packages/api/src/models/index.ts?

File: CHANGELOG.md (hunk 1 above)
Comment (RuyiLi) [reply CHANGELOG.md:9]: Should we merge #214 first? The PaymentOptions tests need to be rewritten with the new ctor

Comment (amcintosh) [reply CHANGELOG.md:9]: Oh yes, That hasn't been merged yet. No, it's fine then. Let's keep to 2.2.0

File: packages/api/src/models/PaymentOptions.ts (hunk 2 above)
Comment (RuyiLi) [reply packages/api/src/models/PaymentOptions.ts]: Done 👍

**Review by amcintosh — APPROVED** [review]

**Pull request merged**